contrib/update_browser_readme_version.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+
+# Paste the version string into the README.
+VERSION=`node -e 'console.log(require("../lerna.json").version)'` 
+VERSION_MAJOR=$(echo $VERSION | cut -d. -f1)
+VERSION_MINOR=$(echo $VERSION | cut -d. -f2)
+VERSION_PATCH=$(echo $VERSION | cut -d. -f3 | cut -d- -f1)
+
+echo "Setting README version to $VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH..."
+
+# Replace the occurences of stitch-sdks/js/bundles/<SOME VERSION>/stitch
+# with stitch-sdks/js/bundles/<CURRENT VERSION>/stitch
+sed -i '' \
+  -e "s#\(stitch-sdks/js/bundles/\)[0-9\.]*\(/stitch\)#\1$VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH\2#g" \
+  ../packages/browser/sdk/README.md
+echo "README updated."
```

Comment:
[minor] To make sure this gets committed before a publish you should add 

```
git add ../packages/browser/sdk/README.md
git commit -m "Update SDK version to $VERSION_MAJOR.$VERSION_MINOR.$VERSION_PATCH in relevant READMEs"
```

Reply:
I'm not sure how I feel about adding git commands in a script... Wouldn't this violate the principle of least astonishment? I wouldn't expect a script to make a commit on my behalf.

Reply:
I put the reminder to commit in the contrib/README.md. Thoughts?